eclass/llvm-utils.eclass

```diff
@@ -150,4 +150,19 @@ llvm_prepend_path() {
 	export PATH=${new_path[*]}
 }
 
+# @FUNCTION: llvm_has_musl_libc
```

Comment:
Okay, so:

1. Technically, a "musl target" check seems something generic enough not to belong to this eclass. However, making it a CMake-specific thing might tilt it closer to "okay".
2. The documentation doesn't tell us anything about the output value.
3. The function name is wrong — "has" suggests it returns a "boolean", i.e. exit status 0/1. What it does instead is print CMake-style ON/yes/no.
4. Do we really need `usex elibc_musl` fallback here? I mean, `${CTARGET:-${CHOST}} == *-musl*` should do exactly that, shouldn't it?

Reply:
I'm happy to move the helper somewhere else and make it more generic. Any ideas about which eclass should it belong to?

Regarding the point 4, `${CTARGET:-${CHOST}} == *-musl*` done in ebuild would work, but I wanted to avoid copying it over. If you feel strongly against having it in any eclass, I can do that too.

Reply:
Let's keep it here but update it per the comments, and figure out some name that would express its purpose better, like `llvm_cmake_usex_musl` or something like that.

Reply:
Done